### PR TITLE
Update CuratorZookeeperRepositoryTest.java

### PR DESCRIPTION
**Pull request by DivvSaxena**
Added Unit Test

Fixes #ISSUSE_ID.

Changes proposed in this pull request:
  -
  -
  -

---

Before committing this PR, I'm sure that I have checked the following options:
- [ ] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [ ] I have self-reviewed the commit code.
- [ ] I have passed maven check: `mvn clean install -B -T2C -DskipTests -Dmaven.javadoc.skip=true -e`.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding unit tests for my changes.

### CODE REVIEW

**Comment by DivvSaxena** [comment]
How can i fix this issue as i added test unit

**Comment by DivvSaxena** [comment]
Is anything wrong?

**Comment by terrymanu** [comment]
Please fix the CI and resolve the conflicts

**Comment by DivvSaxena** [comment]
> Please fix the CI and resolve the conflcts 

Can you help me to fix it as i am not fix it?

File: mode/type/cluster/repository/provider/zookeeper-curator/src/test/java/org/apache/shardingsphere/mode/repository/cluster/zookeeper/CuratorZookeeperRepositoryTest.java [hunk 1]
```diff
@@ -304,4 +304,12 @@ public void assertDeleteExistKey() throws Exception {
...
         verify(backgroundVersionable).forPath("/test/children/1");
     }
+    
+    @Test
+    public void assertUpdate() throws Exception {
+        when(existsBuilder.forPath("/test")).thenReturn(new Stat());
+        REPOSITORY.persist("/test", "value2");
+        verify(setDataBuilder).forPath("/test", "value2".getBytes(StandardCharsets.UTF_8));
+    }
+
```
Comment (linghengqian) [inline mode/type/cluster/repository/provider/zookeeper-curator/src/test/java/org/apache/shardingsphere/mode/repository/cluster/zookeeper/CuratorZookeeperRepositoryTest.java:314]: - What is the point of this PR? A test with the same name already exists.

**Pull request closed without merging**